runtime/ops/tty.rs

```diff
@@ -91,20 +90,9 @@ fn op_set_raw(state: &mut OpState, args: SetRawArgs) -> Result<(), AnyError> {
       return Err(not_supported());
     }
 
-    if resource.fs_file.is_none() {
-      return Err(bad_resource_id());
-    }
-
-    let handle_result = if let Some((fs_file, _)) = &resource.fs_file {
-      let file = fs_file.as_ref().unwrap().clone();
-      let std_file = file.lock().unwrap();
-      let raw_handle = std_file.as_raw_handle();
-      Ok(raw_handle)
-    } else {
-      Err(resource_unavailable())
-    };
-
-    let handle = handle_result?;
+    let std_file = resource.std_file();
+    let std_file = std_file.lock().unwrap(); // hold the lock
```

Comment:
I think it's more correct to hold the lock here if use of this is meant to be exclusive? Not sure though.